clients/src/main/java/org/apache/kafka/common/security/JaasContext.java

```diff
@@ -100,13 +100,8 @@ private static JaasContext defaultContext(JaasContext.Type contextType, String l
                                               String globalContextName) {
         String jaasConfigFile = System.getProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM);
         if (jaasConfigFile == null) {
-            if (contextType == Type.CLIENT) {
-                LOG.debug("System property '" + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' and Kafka SASL property '" +
-                        SaslConfigs.SASL_JAAS_CONFIG + "' are not set, using default JAAS configuration.");
-            } else {
-                LOG.debug("System property '" + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' is not set, using default JAAS " +
-                        "configuration.");
```

Comment:
I am not entirely sure about this. I think that `SASL_JAAS_CONFIG` should be prefixed with the SASL mechanism otherwise we ignore it. In this case, we log a warning in `loadServerContext`. I suppose this is the reason why we don't use a generic message for both cases here. It does not make sense to say that `SASL_JAAS_CONFIG`  is not set if we don't use it without the prefix.